cpp/src/aztec/honk/composer/composer_helper/composer_helper.cpp

```diff
@@ -103,8 +103,11 @@ std::shared_ptr<waffle::verification_key> ComposerHelper<CircuitConstructor>::co
             auto selector_poly_commitment = commitment_key.commit({ selector_poly_coefficients, proving_key->n });
             circuit_verification_key->constraint_selectors.insert(
                 { selector_commitment_label, selector_poly_commitment });
-
-        } else if (selector_poly_info.source == waffle::PolynomialSource::PERMUTATION) {
+            // TODO(luke): Adding commitments to polys with label OTHER to 'permutation_selectors' for now. In the
```

Comment:
As we discussed, this is a temporary measure. We will aim for more uniform handling of the polynomials.